lib/rules/jsx-no-script-url.js

```diff
@@ -77,16 +75,17 @@ module.exports = {
         },
         required: ['name', 'props'],
         additionalProperties: false,
+        deprecated: true, // ?
       },
     }],
   },
 
   create(context) {
-    const config = context.options[0] || [];
+    const linkComponents = context.options[0] ? parseLegacyOption(context.options[0]) : linkComponentsUtil.getLinkComponents(context);
```

Comment:
i think that perhaps this should just always start with the setting, and concatenate the option when present?